deep_qa/training/trainer.py

```diff
@@ -31,7 +31,7 @@ class Trainer:
     The intended use of this class is that you construct a subclass that defines a model,
     overriding the abstract methods and (optionally) some of the protected methods in this class.
     Thus there are four kinds of methods in this class: (1) public methods, that are typically only
-    used by ``scripts/run_model.py`` (or some other driver that you create), (2) abstract methods
+    used by ``scripts/run.py`` (or some other driver that you create), (2) abstract methods
```

Comment:
Here's another place where you need to be consistent between `run_model.py` and `run.py`